modules/preview-react/table/lib/TableHead.tsx

```diff
@@ -1,20 +1,22 @@
 import React from 'react';
 import {GridProps, mergeStyles} from '@workday/canvas-kit-react/layout';
 import {createComponent} from '@workday/canvas-kit-react/common';
-import {createStyles} from '@workday/canvas-kit-styling';
+import {createStencil} from '@workday/canvas-kit-styling';
 import {base} from '@workday/canvas-tokens-web';
 
-const headStyles = createStyles({
-  display: 'grid',
-  backgroundColor: base.soap100,
-  minHeight: '3rem',
-  boxSizing: 'border-box',
-  'th ': {
-    '&:first-of-type': {
-      borderInlineStart: 'none',
-    },
-    '&:last-of-type': {
-      borderInlineEnd: 'none',
+const tableHeadStencil = createStencil({
+  base: {
+    display: 'grid',
+    backgroundColor: base.soap100,
+    minHeight: '3rem',
```

Comment:
this isn't a token huh?

Reply:
Not at 3rem no. 

It goes 1, 2, 3, 4, 6, 8, 10, 16, 20.

Which is: 0.25rem, 0.5rem, 0.75rem, 1rem, 1.5rem, 2rem, 2.5rem, 4rem, 5rem